float64[] po_x
float64[] po_y 
float64[] po_z

float64[] ori_x
float64[] ori_y
float64[] ori_z
float64[] ori_w
